frontend/src/components/Forms/UpdateMineManagerForm.js

```diff
@@ -14,13 +15,18 @@ const propTypes = {
   parties: PropTypes.object.isRequired,
   partyIds: PropTypes.array.isRequired,
   title: PropTypes.string.isRequired,
+  initialValues: PropTypes.object,
 };
 
 const defaultProps = {
   parties: {},
   partyIds: [],
 };
 
+const validateStartDate = memoize(previousStartDate => value => 
+  value <= previousStartDate ? "New manager's start date cannot be on or before the previous manager's start date." : undefined
```

Comment:
@mmumby would `validateStartDate` be better off in a helper/util file?

Personal opinion here: if the `value` just needs to be an error string or a falsey value, I think `false` would make a bit more sense than `undefined`. The reason being that the error "check" would return false. But Meredith may be able to speak to an existing pattern here as well.

Reply:
`validateStartDate` should be in kept in utils/Validate.js as It can be reused in the future.

Reply:
Thanks, Meredith. @james-hollinger having looked at the file Meredith mentioned, the `undefined` is in line with the existing validation functions. So, I revoke my suggestion to change it.

Reply:
Also, please update the tests in Validate.spec.js to reflect this new validate function. You should be able to follow the syntax of the other tests, If you need any help, let me know!

Reply:
That sounds like the best option to me, @james-hollinger

Reply:
Do you have any suggestions that are both clear and concise?

Reply:
We won't know where/when we'll want to reuse this validation, so I'm not sure choosing a general + helpful message is practical yet. The benefit now is keeping the validation functions all in the same spot.

So, I think the message is fine as-is for the time being. Moving the function over should be enough

Reply:
Added front-end unit tests